pkg/controller/azure/azure.go

```diff
@@ -81,6 +89,22 @@ func (c *Controllers) SetupWithManager(mgr ctrl.Manager) error { // nolint:gocyc
 		return err
 	}
 
+	// if err := (&postgresqlservervirtualnetworkrule.Controller{}).SetupWithManager(mgr); err != nil {
+	// 	return err
+	// }
```

Comment:
Should this be commented out?

Reply:
It should not. Forgot to bring that back after ironing out the controller.